compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/programfile/ProgramFileConstants.java

```diff
@@ -25,9 +25,9 @@
 
     public static final int MAGIC_NUMBER = 0xBA1DA4CE;
     public static final short VERSION_NUMBER = 50;
-    public static final int BIR_VERSION_NUMBER = 64;
-    public static final short MIN_SUPPORTED_VERSION = 64;
-    public static final short MAX_SUPPORTED_VERSION = 64;
+    public static final int BIR_VERSION_NUMBER = 65;
```

Comment:
@pubudu91 wouldn't this have an impact on modules built on previous patch releases? (for example, if an external user had published a module to central with a previous patch version of swan lake, would that module work with the next patch release?)

Reply:
Ah sorry, this PR should go to master. Not the patch branch. We are doing BIR changes in master right? @dulajdilshan please note.

Reply:
Changed

Reply:
Still wouldnt this be a major change for someone who is migrating to update1?

Reply:
Do you mean the same as your first comment, but from patch to updates? We get only the source from central.

I think the BIR will/should just get regenerated. @hasithaa mentioned some time back that there was a discussion on if we can do BIR changes in updates and that the conclusion was that we can. Looping in @azinneera and @hevayo also.

Reply:
Only the source is shared. We have a cache directory per distribution. So a new distribution will create its own bir and jar.

E.g: 
Dist 2201.0.1 cache location: `~/.ballerina/repositories/central.ballerina.io/cache-2201.0.1`
Dist 2201.0.2 cache location: `~/.ballerina/repositories/central.ballerina.io/cache-2201.0.2`